crates/edr_provider/src/requests/hardhat/rpc_types.rs

```diff
@@ -0,0 +1,7 @@
+mod compiler;
```

Comment:
Would just `rpc` not suffice for the module name?

Reply:
`rpc_types` is clearer to me, because `rpc` would be for me the entire `requests` module. Happy to rename it though

Reply:
Feel free to leave it